pom.xml

```diff
@@ -663,6 +663,13 @@
         <jackrabbit-core.version>2.5.2</jackrabbit-core.version>
         <jackson1.version>1.9.2</jackson1.version>
         <logback.version>1.2.3</logback.version>
+
+        <scala.base>2.12</scala.base>
+        <scala.version>${scala.base}.8</scala.version>
+        <scala-java8_compat.version>0.9.0</scala-java8_compat.version>
```

Comment:
we use properties when we have at least 2 usage of a value

Reply:
there is 2 usage of scala-java8_compat.version.
One in event/json/pom.xml, the other one in server/task/pom.xml

Reply:
it's not the way it's supposed to be done. You should declare the dependency with the version in dependencyManagement at top-level. That way, the version is declared only once.